src/include/rbd/librbd.h

```diff
@@ -27,22 +27,24 @@ extern "C" {
 #endif
 #include <stdbool.h>
 #include <string.h>
+#include <sys/uio.h>
 #include "../rados/librados.h"
 #include "features.h"
 
-#define LIBRBD_VER_MAJOR 0
-#define LIBRBD_VER_MINOR 1
-#define LIBRBD_VER_EXTRA 11
+#define LIBRBD_VER_MAJOR 1
+#define LIBRBD_VER_MINOR 12
```

Comment:
Ah, I suppose there is no actual version major bump, LIBRBD_VER_MAJOR 0 just was a bug and now it matches librbd SOVERSION. BTW, shouldn't the library VERSION (which is 1.0.0 currently) match major.minor.extra?